tests/util/rpc.py

```diff
@@ -2,17 +2,24 @@
 from chia.rpc.rpc_server import RpcApiProtocol
 
 
-async def validate_get_routes(client: RpcClient, api: RpcApiProtocol):
+async def validate_get_routes(client: RpcClient, api: RpcApiProtocol) -> None:
     routes_client = (await client.fetch("get_routes", {}))["routes"]
     assert len(routes_client) > 0
     routes_api = list(api.get_routes().keys())
+    # TODO: avoid duplication of RpcServer.get_routes()
     routes_server = [
         "/get_connections",
         "/open_connection",
         "/close_connection",
         "/stop_node",
         "/get_routes",
+        "/healthz",
     ]
     assert len(routes_api) > 0
-    for route in routes_api + routes_server:
-        assert route in routes_client
+
+    all_server_routes = {*routes_api, *routes_server}
+    all_client_routes = {*routes_client}
+    only_server = sorted(all_server_routes - all_client_routes)
+    only_client = sorted(all_client_routes - all_server_routes)
+
+    assert [only_server, only_client] == [[], []]
```

Comment:
```suggestion
    assert {*routes_client} == {*routes_api, *routes_server}
```

Any reason to not just do the above?

Reply:
or maybe rather:

```suggestion
    assert sorted(routes_client) == sorted(routes_api + routes_server)
```